### PR TITLE
Add `spotPrice` on `Offer`.

### PR DESCRIPTION
**Pull request by iaronaraujo**
#### What problem is this solving?

As the title says.

#### How should this be manually tested?

[Workspace](https://productprice--storecomponents.myvtex.com/fashion-eyeglasses/p)
Check the spot price below the selling price.

#### Screenshots or example usage

image

### CODE REVIEW

File: node/resolvers/search/product.ts [hunk 1]
```diff
@@ -219,4 +219,15 @@ export const resolvers = {
   OnlyProduct: {
     categoryTree: productCategoriesToCategoryTree,
   },
+  Offer: {
+    spotPrice: (offer: CommertialOffer) => {
+      let spotPrice = offer.Price
+      offer?.Installments.forEach(({NumberOfInstallments, Value}) => {
+        if (NumberOfInstallments === 1 && Value < spotPrice) {
+          spotPrice = Value
+        }
```
Comment (kaisermann) [inline node/resolvers/search/product.ts:228]: Do you need to iterate through all the installments or only the first that matches?

Comment (iaronaraujo) [reply node/resolvers/search/product.ts:228]: TBH I am not sure, so I decided to iterate through all of them to be safe, but I can check that!

Comment (kaisermann) [reply node/resolvers/search/product.ts:228]: 🤷

Comment (klzns) [reply node/resolvers/search/product.ts:228]: Need to get the lowest value, so it needs to iterate through all

**Pull request merged**